src/hotspot/share/jfr/periodic/jfrOSInterface.cpp

```diff
@@ -32,8 +32,6 @@
 #include "runtime/vm_version.hpp"
 #include "utilities/ostream.hpp"
 
-#include <stdlib.h> // for environment variables
```

Comment:
I couldn't find any usage for this include, let me know if it should stay. It's been here for a long time: https://github.com/openjdk/jdk/commit/a060be188df894ed5c26fc12fc9e902f9af32bd3

Reply:
Sure: f734135f2ecc399ef2aa2a2d150ebba454e5776d

https://bugs.openjdk.org/browse/JDK-8365782